### ISSUE TITLE
External source map not getting generated with correct file name

### ISSUE THREAD

**Issue by balajisivanath** [tivac/modular-css#455]
Given the following rollup configuration, external css source map is getting generated with file name as "bundle2.css" instead of "bundle.css.map"
```
import css from "modular-css-rollup";
export default {
    input: "source/exports.js",
    output: {
        file: "out/bundle.js",
        format: "iife",
        name: "test",
        sourcemap: true,
        assetFileNames: "bundle.css" //If this is not given, then modular-css-rollup outputs the file assets/bundle-[hash].css
    },
    plugins: [
        css({
            map: {
                inline: false
            }
        })
    ]
};
```
![annotated](https://user-images.githubusercontent.com/2368442/42864074-cb44553a-8a82-11e8-8dde-364a2e9ba5f1.png)

Rollup version: 0.62.0
Modular css version: 14.0.1

**Comment by tivac** [comment]
What happens if you use `assetFileNames : "[name][extname]"` instead?

It's very similar to what I do in the tests

https://github.com/tivac/modular-css/blob/5f1edd68abd2d2a3b1f408f67f4bd4bc0a25006c/packages/rollup/test/rollup.test.js#L24

and that looks to generate the expected name based on

https://github.com/tivac/modular-css/blob/5f1edd68abd2d2a3b1f408f67f4bd4bc0a25006c/packages/rollup/test/rollup.test.js#L219-L243

**Comment by balajisivanath** [comment]
> What happens if you use assetFileNames : "[name][extname]" instead?

Changing to this format generates file correctly 👍

**Comment by tivac** [comment]
Weirdly I don't see anywhere in my code that is generating the `2.css` version containing the map code, though I can replicate what you're seeing.

I'd argue that the original config value being a hardcoded filename is an incorrect value to pass, but ideally it wouldn't explode things... 😥

**Comment by balajisivanath** [comment]
Ouch! Sourcemapping url in bundle.css isn't replaced with correct filename
![image](https://user-images.githubusercontent.com/2368442/42864807-47e474ec-8a85-11e8-9b35-ac1dfe6bc93d.png)

> Weirdly I don't see anywhere in my code that is generating the 2.css version containing the map code, though I can replicate what you're seeing.
I'd argue that the original config value being a hardcoded filename is an incorrect value to pass, but ideally it wouldn't explode things... 😥

I guess issue could be in rollup, considering that `assetFileName` config is under their experimental code split feature

**Comment by tivac** [comment]
Well that `sourceMappingURL` is definitely a problem. Ugh, the interactions between PostCSS and rollup are a real thing sometimes.

I can fix that one, at least!

**Comment by balaji-sivanath** [comment]
@tivac Can you please tag a new release with this fix?
Thanks in advance.

**Comment by tivac** [comment]
Latest release should have this change afaik. Let me know if it doesn't!

**Comment by balajisivanath** [comment]
Latest release v14.1.0 is built with this changeset 119e10c. Above fix is merged after that, hence it is not there.

**Comment by tivac** [comment]
Hrmm, not sure how that missed the 14.1.0 release since I intended it to be part of it. My mistake, I'll try to get to that soonish.

**Comment by tivac** [comment]
Released in `modular-css-rollup@14.2.0`, sorry about the delay!
